internal/vehicle/ford.go

```diff
@@ -134,36 +169,135 @@ func (v *Ford) vehicles() ([]string, error) {
 	return vehicles, err
 }
 
-// chargeState implements the api.Vehicle interface
-func (v *Ford) chargeState() (float64, error) {
-	var resp struct {
-		VehicleStatus struct {
-			Battery struct {
-				BatteryStatusActual struct {
-					Value int
-				}
-			}
-			BatteryFillLevel struct {
-				Value float64
+// calculateAge parses Ford API timestamp and returns age
+func (v *Ford) calculateAge(timestamp string) (age time.Duration, err error) {
```

Comment:
Ich würde immer noch die 2 Zeilen inline machen, Format ggf als Konstante:

```
func (v *Ford) age(timestamp string) (time.Duration, error) {
	ts, err := time.Parse("01-02-2006 15:04:05", timestamp) // UTC timestamp
	return time.Since(ts), err
}
```